src/plugins/action_update_review_status/src/updateReviews.ts

```diff
@@ -0,0 +1,70 @@
+import databaseMethods from './databaseMethods'
+
+type ReviewStatus = 'Draft' | 'Submitted' | 'Changes Requested' | 'Pending' | 'Locked'
+interface Review {
+  reviewId: number
+  reviewAssignmentId: number
+  applicationId: number
+  reviewerId: number
+  level: number
+  reviewStatus: ReviewStatus
+}
+
+module.exports['updateReviews'] = async function (input: any, DBConnect: any) {
+  const db = databaseMethods(DBConnect)
+
+  console.log('Updating reviews status...')
+
+  const { applicationId, changedApplicationResponses = [] } = input
+
+  const reviewsToUpdate = []
+
+  console.log('changedApplicationResponses', changedApplicationResponses)
+
+  try {
+    // Get reviews/review assignments (with status) matching application_id
+    const reviews = (await db.getAssociatedReviews(applicationId)).filter((review: Review) =>
+      ['Submitted', 'Locked', 'Draft'].includes(review.reviewStatus)
```

Comment:
kinda like the idea of filtering with postgresql, but this looks fine (don't need to change).

Reply:
Yeah, I think I was thinking that the database method might be more re-usable if if didn't have additional filtering -- but it's associated with this Action only, so doesn't really matter.